manifests/graphite_reqs.pp

```diff
@@ -49,7 +49,7 @@
   }
   package { 'python-sqlite3dbm':
     ensure => '0.1.4-6.el7',
-    provider => 'pip',
+    provider => 'yum',
```

Comment:
you shouldn't need to set it at all. It will default to yum on a centos machine.

And if you *do* set it, that makes your code really difficult to port to another platform if we ever decide to do that.